accounts/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Can you add some testing maybe? This is quite a complex flow and would benefit from adding at least some simple tests.

Reply:
Going to add this as a separate issue.